### PR TITLE
Unit tests for local server

### LINKED ISSUE #7482: Write unit tests for `src/server/local/index.js`
**Is your feature request related to a problem? Please describe.**
In [this refactor PR](https://github.com/bbc/simorgh/pull/7279/files), the `src/server/local/index.js` file was created, but no unit tests were added. 
As a result, as part of this PR: https://github.com/bbc/simorgh/pull/7391 we seemed to have decreased test coverage below the 80% threshold for the diff of the PR. The comscore unit tests have 100% line coverage, and we noticed that  `src/server/local/index.js` had no test file. 

**Describe the solution you'd like**
We should write some unit tests to cover this file:  `src/server/local/index.js`

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by karinathomasbbc**
Resolves #7482

**Overall change:**
Adding unit tests for src/server/local/index.js

**Code changes:**
- Re-enabled Africa Eye TV data path
- Add unit tests for local server

---

- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] I have added the `cross-team` label to this PR if it requires visibility across World Service teams
- [ ] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/server/local/index.test.js [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+    articleDataPaths.forEach(articleDataPath => {
+      assertDataFile({
+        dataPath: articleDataPath,
+        expectedDataFile: articleDataPath,
+      });
+    });
+  });
+
+  describe('Front Page data file', () => {
+    const frontPageDataPaths = [
```
Comment (chris-hinds) [inline src/server/local/index.test.js]: Most of these tests are very similar. Could we use the `it.each` logic to reduce the number of repeated tests?

**Review by chris-hinds — CHANGES_REQUESTED** [review]

File: src/server/local/index.test.js (hunk 1 above)
Comment (chris-hinds) [reply src/server/local/index.test.js]: Similar to tests here `/src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/buildIncludeUrl/index.test.js`

Comment (karinathomasbbc) [reply src/server/local/index.test.js]: Done! Thanks for the signpost, it was very helpful 😄

**Review by chris-hinds — APPROVED** [review]
Tests look ace, much more readable now :) 👍

**Review by andrewscfc — APPROVED** [review]

**Pull request merged**